ci/release/changelogs/next.md

```diff
@@ -12,3 +12,4 @@
 - Fixes styles in connections not overriding styles set by globs [#1857](https://github.com/terrastruct/d2/pull/1857)
 - Fixes `null` being set on a nested shape not working in certain cases when connections also pointed to that shape [#1830](https://github.com/terrastruct/d2/pull/1830)
 - Fixes edge case of bad import syntax crashing using d2 as a library [#1829](https://github.com/terrastruct/d2/pull/1829)
+- Fixes `style.fill` not applying to markdown [#1872](https://github.com/terrastruct/d2/pull/1872)
```

Comment:
it's `style.font-color` right?

Reply:
`style.font-color` was just updating the test case (we used `style.stroke` for font-color when the test was written). The functional change I added was to restore the part which applies `style.fill` to set the background color.